From ff629766dbd3b5e179330ba4f07fd7e3d8384ec0 Mon Sep 17 00:00:00 2001
From: Max Filippov <jcmvbkbc@gmail.com>
Date: Wed, 1 Nov 2017 15:30:40 +0100
Subject: [PATCH] configure.ac: fix AC_CHECK_HEADER call to test for libssh2

The -lm argument is passed as an argument to AC_CHECK_HEADER(), which
doesn't make sense. The intention was to pass it as the fifth
argument of AC_CHECK_LIB().

Signed-off-by: Max Filippov <jcmvbkbc@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 3d00e164e..67db42e08 100644
--- a/configure.ac
+++ b/configure.ac
@@ -674,7 +674,7 @@ AC_HELP_STRING([--without-libssh2], [Compile without libssh2]),
       AC_CHECK_LIB(ssh2, libssh2_version,
         [have_libssh2=yes
         LIBSSH2_INC=$with_libssh2/include
-        LIBSSH2_LIB=$with_libssh2/lib])],,[-lm])
+        LIBSSH2_LIB=$with_libssh2/lib],,[-lm])])
 
     LDFLAGS=$_ldflags
     CPPFLAGS=$_cppflags
-- 
2.13.6

